lib/app_profiler.rb

```diff
@@ -28,8 +28,8 @@ module Storage
 
   module Viewer
     autoload :BaseViewer, "app_profiler/viewer/base_viewer"
-    autoload :SpeedscopeViewer, "app_profiler/viewer/speedscope_viewer"
-    autoload :SpeedscopeRemoteViewer, "app_profiler/viewer/speedscope_remote_viewer"
```

Comment:
If I'm reading this right, we are removing the remote viewer. It is public API, so let's revert that.

Reply:
@gmcgibbon as the PR description says, the two are consolidated. What was formerly the "remote" viewer is now the only option as I don't see any benefit to having two ways to do the same thing.

I understand it's a public API change, i think releasing as 0.2.0 with a note would make sense.

Likewise, it also removes the viewer being configurable (AppProfiler.viewer config will just be ignored) - stackprof will use speedscope, vernier will use firefox-profiler.